pages/docs.json

```diff
@@ -62,6 +62,13 @@
           "title": "TypeScript"
         }
       ]
+    }, {
+      "title": "FAQ's and Tips & Tricks",
+      "pathname": "faqs",
```

Comment:
Can we rename this to just be `faq` please? :) Also above, we can just use "FAQs" (no apostrophe)